### PR TITLE
Added package description

### PR DESCRIPTION
**Pull request by MarcinOrlowski**
Adds `--description` option to jpackage to improve built packages. 

Before:

before

After:

after

BTW: Anyone knows why we have `--file-associations` in linux specific options not in shared ones?

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -67,7 +67,9 @@ extra.apply {
       "--main-jar", "${project.name}-${project.version}-all.jar",
       "--app-version", project.version as String,
       "--copyright", "Copyright © 2001–" + year + " Carl Burch, BFH, HEIG-VD, HEPIA, Holy Cross, et al.",
-      "--dest", "${buildDir}/dist"
+      "--dest", "${buildDir}/dist",
+      "--description", "Digital logic designer and simulator",
```
Comment (BFH-ktt1) [inline build.gradle.kts:71]: "Digital logic designer and simulator" hears like it designs itself, I would change it in:
"Digital logic design tool and simulator", or something alike

Comment (davidhutchens) [reply build.gradle.kts:71]: I agree "design tool" sounds better.

**Comment by davidhutchens** [comment]
Each platform has its own file-associations. Those provide platform specific information such as location of icon files (which are in different formats with different extensions).

File: build.gradle.kts (hunk 1 above)
Comment (MarcinOrlowski) [reply build.gradle.kts:71]: Fixed but:

ss

**Review by davidhutchens — APPROVED** [review]
That looks good to me. But @MarcinOrlowski makes a good point about the About on our main page.

File: build.gradle.kts (hunk 1 above)
Comment (BFH-ktt1) [reply build.gradle.kts]: I fixed that one :-)

**Review by BFH-ktt1 — APPROVED** [review]

**Comment by maehne** [comment]
@davidhutchens: I am not sure I got fully your point regarding the "About" on our main page, because I did not find a matching comment by @MarcinOrlowski in this PR. Nevertheless, I updated the About block to "Digital design tool and simulator" to match the package description added in this PR.

**Comment by davidhutchens** [comment]
@maehne: The comment regarding "About" referred to the resolved communication in @BFH-ktt1's first review above.

**Pull request merged**